robottelo/ui/locators/tab.py

```diff
@@ -28,9 +28,11 @@
     "host.tab_puppet_classes": (By.XPATH, "//a[@href='#puppet_klasses']"),
     "host.tab_interfaces": (By.XPATH, "//a[@href='#network']"),
     "host.tab_operating_system": (
-        By.XPATH, "//form[@id='new_host']//a[@href='#os']"),
+        By.XPATH, ("//form[@id='new_host' or contains(@id,'edit_host')]"
```

Comment:
Yeah.. so when we create a host from `All hosts -> New Host` then form id value set as `new_host`.

However when we click on `provision` from `discoveredhost` page then `Edit Host` page open up and so id value set as `edit_host` and  I don't want to add new locator with this small change.